package.json

```diff
@@ -15,6 +15,7 @@
     "lint": "eslint . --ext ts,tsx --report-unused-disable-directives --max-warnings 0",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "vite --host || echo this echo is here to ignore the --config arg if tutor passes it in",
+    "start-tutor": "PUBLIC_PATH=/profile/ MFE_CONFIG_API_URL='http://localhost:8000/api/mfe_config/v1' vite --host",
```

Comment:
I'm _very_ interested in getting something like this working _before_ (and if) we migrate to Vite, haha.  I'll give it a shot and let you know.

Reply:
It works! I use it in course-authoring all the time. It's much easier and faster than running it inside a tutor container.

The command is:
```
APP_ID=course-authoring PUBLIC_PATH=/course-authoring/ MFE_CONFIG_API_URL='http://localhost:8000/api/mfe_config/v1' npm run start -- --host apps.local.overhang.io
```

And honestly the MFEs should be fixed so these env vars are the defaults, which would simplify this a lot.